README.md

```diff
@@ -1,42 +1,126 @@
+
+4. Check our [Toolbelt design guidelines](https://github.com/vtex/ux-writing/blob/new-content/docs/text-patterns/toolbelt-cli.md) for keeping up with the **best practices** when developing your plugin.
+
+5. Check [oclif](https://oclif.io/) basic concepts.
+
+>ℹ️ Toolbelt is built with [oclif](https://oclif.io/docs/introduction), a framework for building extensible CLIs in Node.
+
+### Setting up the local development environment
+
+To locally test your plugins as you code, you'll need to use the `yarn watch` command, which is responsible for creating a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) named `vtex-test`.
+
+To use the `yarn watch` command, you must add `$HOME/.vtex/dev/bin` to your `PATH` environment variable. Take the following steps:
+
+>⚠️ ***Keep in mind:** these steps may vary depending on your operating system.*
+
+1. Open the `~/.bashrc` file (or the corresponding shell configuration file).
+2. At the end of this file, add the line: `export PATH=$PATH:$HOME/.vtex/dev/bin`.
+3. Save it.
+4. Run the command source `~/.bashrc` (or corresponding file).
+
+## Creating a new plugin
+
+### Step 1: Creating and organizing your repository
+
+1. [Create a new repository from this template](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) inside the [`vtex` organization.](https://github.com/vtex)
+2. Open the `package.json` and update the `name` and `description` fields, according to your scenario.
+
+>⚠️ *Please keep the standard `@vtex/cli-plugin-{name}` when naming your plugin.*
+
+3. Check if `vtex` under `devdependecies` in the `package.json` file is referring to the [last version of the Toolbelt](https://github.com/vtex/toolbelt/releases) and update it if necessary.
+
+4. Open the `readme.md` file and update it, ensuring that you replace `template` with the name of your plugin and removing any unnecessary content.
+
+>⚠️ *You don't need to worry about updating the [usage](#usage) and [command](#command) references, since these sections will be automatically updated.*
+
+### Step 2: Developing your plugin
+
+Develop your plugin commands inside the `/src/commands/` folder, referring to the [oclif API reference](https://oclif.io/docs/commands) and `/src/commands/hello.ts` example file.
+
+Also, if necessary, feel free to create libraries, clients, and [modules](https://oclif.io/docs/running_programmatically#sharing-code-with-modules) inside your `src` folder.
+
+>ℹ️ *Please refer to our [Toolbelt design guidelines](https://github.com/vtex/ux-writing/blob/new-content/docs/text-patterns/toolbelt-cli.md) for keeping up with the **best practices** when developing your plugin.*
+
+### Step 3: Testing your plugin
+
+1. Run `yarn link` in your plugin project.
+2. Now, in the **cloned** Toolbelt directory, run `vtex link @vtex/cli-plugin-{name}`.
+3. Run `yarn watch` in both the cloned `vtex/toolbelt` and in the plugin directory. 
```

Comment:
How about use the code syntax to "plugin directory" ? 

plugin directory => `plugin directory`

Reply:
Also change "directory" to "repo" or "repository"

`plugin directory` => `plugin repo`